### ISSUE TITLE
Tabbable element role ‘group’ is not widget role

### ISSUE THREAD

**Issue by samsadasivan** [cds-snc/platform-forms-client#5833]
**_Specs: Windows
Screen Readers Used: JAWS, NVDA
Browsers Tested: Google Chrome, Microsoft Edge, Internet Explorer_**

**Observation:** 
Issues observed in the “Date of Birth” and “Date of Submission” fields. 
The tabbable element's role 'group' is not a widget role. A tabbable element must have a valid widget role. 

A keyboard-focusable (tabbable) element has been found with an invalid role. The only elements that should be in the tab order should be operable widgets. Inoperable elements, such as headings, can be given focus; however, they should not be tabbable by the user. 

**_Severity: Critical_** 

**Recommendation:**  

If users should not be able to interact with this element using the keyboard, replace the non-widget role, and/or use the explicit tabindex="-1" attribute to remove the element from the keyboard focus (tab) order 

Date of birth :  

<fieldset role="group" data-testid="formattedDate" id="6" tabindex="0"> 

Date of submmission: 

 <fieldset role="group" data-testid="formattedDate" id="21" tabindex="0">

**Comment by samsadasivan** [comment]
This part is not updating in the Markdown for the description under "Recommendation" 

<img width="1526" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBenFnQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5dbf96c0a3b8b27d2e7890ab0aa95f5c3f01f78d/image.png" alt="image.png" />

**Comment by samsadasivan** [comment]
Date of birth :  

<fieldset role="group" data-testid="formattedDate" id="6" tabindex="0"> 

Date of submmission: 

 <fieldset role="group" data-testid="formattedDate" id="21" tabindex="0">
